airbyte-ci/connectors/pipelines/pipelines/consts.py

```diff
@@ -13,14 +13,11 @@
 CONNECTOR_TESTING_REQUIREMENTS = [
     "pip==21.3.1",
     "mccabe==0.6.1",
-    "flake8==4.0.1",
-    "pyproject-flake8==0.0.1a2",
-    "black==22.3.0",
-    "isort==5.6.4",
+    # "flake8==4.0.1",
+    # "pyproject-flake8==0.0.1a2",
```

Comment:
Are any connector tests running linting right now? Am I missing something?

Reply:
This might date back to when python connectors were built using gradle. This pip package inclusion list looks very similar to what used to be in the airbyte-python gradle plugin. IIRC you can remove mccabe and coverage[toml] as well but I'm not sure.